lib/platform/azure/index.ts

```diff
@@ -315,7 +316,7 @@ export async function findPr(
 
 export async function getBranchPr(branchName: string): Promise<Pr | null> {
   logger.debug(`getBranchPr(${branchName})`);
-  const existingPr = await findPr(branchName, null, 'open');
+  const existingPr = await findPr({ branchName, prTitle: null, state: 'open' });
```

Comment:
`prTitle` is optional so can we remove it from the parameters? I assume we don't check for `null` explicitly within the function itself